Documentation/operations/performance/tuning.rst

```diff
@@ -37,6 +37,55 @@ in any of the Cilium pods and look for the line reporting the status for
 * eBPF-based kube-proxy replacement
 * eBPF-based masquerading
 
+IPv6 BIG TCP
+============
+
+IPv6 BIG TCP allows the network stack to prepare larger TSO (transmit) and GRO
+(receive) packets to reduce the number of times the stack is traversed which
+improves performance and latency. It reduces the CPU load and helps achieve
+higher speeds (i.e. 100Gbit/s and beyond). To pass such packets through the stack
+BIG TCP adds a temporary Hop-By-Hop header after the IPv6 one which is stripped
+before transmitting the packet over the wire. BIG TCP can operate in a DualStack
+setup, IPv4 packets will use the old lower limits (64k) and IPv6 packets will
+use the new larger ones (192k). Note that Cilium assumes the default kernel values
+for GSO and GRO maximum sizes are 64k and adjusts them only when necessary, i.e. if
+BIG TCP is enabled and the current GSO/GRO maximum sizes are less than 192k it
+will try to increase them, respectively when BIG TCP is disabled and the current
+maximum values are more than 64k it will try to decrease them.
+
+**Requirements:**
+
+* Kernel >= 5.19
```

Comment:
Maybe worth adding a line on kernel requirements in https://docs.cilium.io/en/latest/operations/system_requirements/#required-kernel-versions-for-advanced-features as well?